sips/sip-013/sip-013-semi-fungible-token-standard.md

```diff
@@ -0,0 +1,248 @@
+# Preamble
+
+SIP Number: 013
+
+Title: Standard Trait Definition for Semi-Fungible Tokens
+
+Author: Marvin Janssen <https://github.com/MarvinJanssen>
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Draft
+
+Created: 12 September 2021
+
+License: CC0-1.0
+
+Sign-off: 
+
+Layer: Traits
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+Semi-Fungible Tokens, or SFTs, are digital assets that sit between fungible and non-fungible tokens. Fungible tokens are directly interchangeable, can be received, sent, and divided. Non-fungible tokens each have a unique identifier that distinguishes them from each other. Semi-fungible tokens have both an identifier and an amount. This SIP describes the SFT trait and provides a reference implementation.
```

Comment:
Formatting nit -- can you break lines into 80 columns?  It will make merging patches less conflict-prone.